app/controllers/staff/events_controller.rb

```diff
@@ -21,23 +21,36 @@ def speaker_emails
   def guidelines
   end
 
-  def update_guidelines
-    if @event.update(params.require(:event).permit(:guidelines))
-      flash[:info] = 'Your guidelines were updated.'
-      redirect_to event_staff_guidelines_path
-    else
-      flash[:danger] = 'There was a problem saving your guidelines; please review the form for issues and try again.'
-      render :edit
-    end
+  def information
+  end
+
+  def configuration
   end
 
   def update_custom_fields
```

Comment:
I'd love to talk about possible ways to combine the three repetitive custom update methods here.